app/controllers/find/results_controller.rb

```diff
@@ -4,14 +4,16 @@ module Find
   class ResultsController < ApplicationController
     before_action :render_feedback_component
 
+    PAGY_PER_PAGE = 10
```

Comment:
This value is reused quite a bit. It seems that 10 results is the most common amount per page? I haven't checked the other parts, but does it make more sense to set this as the global default for pagy?

Reply:
Wait, wait.. Pagy comes with a way to set default values in the initializer. `Pagy::DEFAULT[:limit] = 10`, I believe. Check their docs :)